scripts/test_model.py

```diff
@@ -0,0 +1,31 @@
+import os
+import pickle
+import click
+from sklearn.metrics import classification_report
+from pkspace.utils.loaders import PKSpaceLoader, PKLotLoader
+
+
+@click.command()
+@click.option('--loader', '-l', type=click.Choice(['PKLot', 'PKSpace']),
+              default='PKSpace', help='Loader used to load dataset')
+@click.argument('dataset_dir')
+@click.argument('model_file')
```

Comment:
@rstevanak Here we can add similar line to the trainer file `output = os.path.join(dataset_dir, 'out.pkl')` so the `click` would take care of handling the existence of these file for us.